variables.tf

```diff
@@ -463,6 +464,12 @@ variable "allow_worker_internet_access" {
   type        = bool
 }
 
+variable "allow_pod_internet_access" {
+  default     = true
+  description = "Allow pods to egress to internet. Required if the pods are invoking Internet services."
```

Comment:
I would perhaps rephrase this as "Allow pod egress to the internet." 

Also, we should check the style guide to see if Oracle refers to "the Internet" or "the internet" and use that form throughout.

Reply:
"internet" wins on the style guide. Also, for future reference https://en.wikipedia.org/wiki/Capitalization_of_Internet